README.md

```diff
@@ -1,30 +1,142 @@
+<h1 align="center"><img src="https://github.com/JanssenProject/jans/blob/eea33ea5c7740244b45f7bb1893167cd3fb93903/docs/logo/janssen_project_transparent_630px_182px.png" alt="Janssen Project - cloud native identity and access management platform"></h1>
+
+## Welcome to the Janssen Project
+
+Janssen enables organizations to build a scalable centralized authentication and authorization service using free open source software. The components of the project include client and server implementations of the OAuth, OpenID Connect, SCIM and FIDO standards. 
+
+**Releases**: [Latest](https://github.com/JanssenProject/jans/releases/latest) | [All](https://github.com/JanssenProject/jans/releases)
+
+**Get Help**: [Chat](https://gitter.im/JanssenProject/Lobby) | [Discussions](https://github.com/JanssenProject/jans/discussions)
```

Comment:
Lets put the discussions first.

Reply:
Fixed with [a72b528](https://github.com/JanssenProject/jans/pull/1055/commits/a72b52838030607e484eeeed2ea372b57c10bb05)